common/observers/fs/fs.go

```diff
@@ -0,0 +1,127 @@
+package fs
+
+import (
+	"bytes"
+	"github.com/skydive-project/skydive/common"
+	"github.com/skydive-project/skydive/logging"
+	"gopkg.in/fsnotify/fsnotify.v1"
+	"io/ioutil"
+	"sync"
+	"sync/atomic"
+	"time"
+)
+
+// WatcherCallback described the interface for callback on matched event
+type WatcherCallback = func(details Event)
+
+// Op mimics fsnotify.Op for backward compatibility with fsnotify
+type Op fsnotify.Op
+
+// Event mimics fsnotify.Event for backward compatibility with fsnotify
+type Event fsnotify.Event
```

Comment:
I do not see where you are using fsnotify and why you have to ? I would define my own event structure to pass the content in the event. It would avoid the read it twice, in the checkFileContentEvents and in the srio-v callback